app/tests/integration/pages/experiment_design.py

```diff
@@ -18,7 +18,7 @@ class DesignPage(Base):
     _firefox_pref_type_locator = (By.CSS_SELECTOR, "#id_pref_type")
     _firefox_pref_branch_locator = (By.CSS_SELECTOR, "#id_pref_branch")
     _new_branch_locator = (By.CSS_SELECTOR, "#add-branch-button")
-    _page_wait_locator = (By.CSS_SELECTOR, ".page-edit-design")
+    _page_wait_locator = (By.CSS_SELECTOR, "body #design-form .form-label h5")
```

Comment:
This is the kind of structural query I'd like to avoid, is there some element on the page you can add a class to and just look that up?

Reply:
Example: I added this here for this same reason: https://github.com/mozilla/experimenter/blob/master/app/experimenter/static/js/components/TimelinePopForm.js#L204

Reply:
Gotcha, I'll do that

Reply:
So I did this because the usual page wait doesn't work. ```is_displayed()``` is checking for a CSS ```display``` property, which this page doesn't have.

Reply:
I don't think its a big deal if we can't use the same page wait. As long as selenium waits correctly. Unless you want to create a react element that changes when the page finishes loading.

Reply:
The react component top level div has a class name:

https://github.com/mozilla/experimenter/blob/master/app/experimenter/static/js/components/DesignForm.js#L192

That is not defined in the template:

https://github.com/mozilla/experimenter/blob/master/app/experimenter/templates/experiments/edit_design.html

So that class name should only become present after react has finished loading, right?  That's what  I did on the timeline page?

Am I missing something?